bot/exts/easter/save_the_planet.py

```diff
@@ -0,0 +1,29 @@
+import json
+import random
+from pathlib import Path
+
+from discord import Embed
+from discord.ext import commands
+
+
+class SaveThePlanet(commands.Cog):
+    """A cog that teaches users how they can help our planet."""
+
+    embed_data = []
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        with open(Path("bot/resources/easter/save_the_planet.json"), 'r', encoding='utf8') as f:
+            for embed in json.load(f):
+                self.embed_data.append(embed)
```

Comment:
```suggestion
    def __init__(self, bot: commands.Bot) -> None:
        self.bot = bot
        self.embed_data = []

        with open(Path("bot/resources/easter/save_the_planet.json"), 'r', encoding='utf8') as f:
            self.embed_data = [embed for embed in json.load(f)]
```
This would be the a perfect place to use a list comprehension.

Reply:
Oh you can do that? Cool, I'll make that change.